solr/core/src/java/org/apache/solr/handler/designer/SchemaDesignerAPI.java

```diff
@@ -87,7 +87,7 @@
 public class SchemaDesignerAPI implements SchemaDesignerConstants {
 
   private static final Set<String> excludeConfigSetNames =
-      new HashSet<>(Arrays.asList(DEFAULT_CONFIGSET_NAME, BLOB_STORE_ID));
+      new HashSet<>(Arrays.asList(DEFAULT_CONFIGSET_NAME));
```

Comment:
This should use `Set.of` as it's declared as a constant and thus needs to be immutable